.golangci.yml

```diff
@@ -0,0 +1,16 @@
+linters:
+  enable:
+    - gofmt
+    - misspell
+    - godot
+    - whitespace
+    - gci
+linters-settings:
+  godot:
+    exclude:
+      - "^ API extension:"
```

Comment:
Can we have a comment saying why?

Reply:
Coming back to this I'm not sure I can really justify it. This was so that there is no need to have a full-stop when specifying the API extension e.g. https://github.com/lxc/lxd/blob/master/shared/api/certificate.go#L41.

There is no particular reason that there can't be a full-stop there, so perhaps we should remove the exclusion. What do you think?

Reply:
I don't mind having a full stop there, is this ok with you and swagger @stgraber ?

Reply:
I've removed the config and added the full stops.